src/analysis/calculus/fderiv.lean

```diff
@@ -1680,6 +1680,62 @@ theorem fderiv_sum (h : ∀ i ∈ u, differentiable_at 𝕜 (A i) x) :
 
 end sum
 
+section pi
+
+/-! ### Derivatives of functions `f : E → Π i, F i` -/
+
+variables {ι : Type*} [fintype ι] {F' : ι → Type*} [Π i, normed_group (F' i)]
+  [Π i, normed_space 𝕜 (F' i)] {φ : Π i, E → F' i} {φ' : Π i, E →L[𝕜] F' i}
+
+lemma has_strict_fderiv_at_pi (h : ∀ i, has_strict_fderiv_at (φ i) (φ' i) x) :
```

Comment:
This lemma and many of the following ones are in fact equivalences: if the product map has a strict derivative, then each component has a strict derivative. Could you formulate them as iffs? (I needed something like that for `pi_Lp` for LFTCM2020 but I never ended up PRing it, see https://github.com/leanprover-community/lftcm2020/blob/master/src/for_mathlib/manifolds.lean)